src/VisualStudio/Setup/VisualStudioSetup.csproj

```diff
@@ -30,98 +30,122 @@
     <ProjectReference Include="..\..\Compilers\Core\Portable\CodeAnalysis.csproj">
       <Project>{1EE8CAD3-55F9-4D91-96B2-084641DA9A6C}</Project>
       <Name>CodeAnalysis</Name>
+      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup</IncludeOutputGroupsInVSIX>
```

Comment:
It doesn't seem to be.  I tried UseCommonOutputDirectory, and it did not achieve the desired effect.